letsencrypt/client/network2.py

```diff
@@ -90,6 +88,9 @@ def _check_response(cls, response, content_type=None):
                         response_ct)
 
                 try:
+                    # TODO: This is insufficient or doesn't work as intended.
```

Comment:
please elaborate; is that because raised `messages2.Error` doesn't print any useful tmessage on CLI when the exception is not caught and traceback stack is printed?